t/perf/opcount.t

```diff
@@ -696,10 +696,19 @@ test_opcount(0, "builtin::true/false are replaced with constants",
                 });
 
 test_opcount(0, "builtin::isbool is replaced with direct opcode",
-                sub { my $x = 123; my $y = builtin::isbool($x); },
+                sub { my $x = 123; my $y; $y = builtin::isbool($x); },
                 {
                     entersub => 0,
                     isbool   => 1,
+                    padsv    => 3, # OA_TARGLEX applies so only 3, not 4
```

Comment:
Should this also check `sassign => 0`?

Reply:
Well, 1 because `$x` still has one. Just not `$y`. But yes I'll add that. 2df1a36e41

Reply:
You could avoid the `$x` one by leaving it uninitialised, since `isbool(undef)` is perfectly valid.

Reply:
Ohyes, I just wanted something that wouldn't const-fold. That will do too. 5ea6acf8e9